terraform/global-resources/auth0.tf

```diff
@@ -5,13 +5,6 @@ resource "auth0_rule" "whitelist-github-orgs" {
   enabled = true
 }
 
-resource "auth0_rule" "whitelist-github-teams" {
-  name    = "whitelist-github-teams"
-  script  = "${file("${path.module}/resources/auth0-rules/whitelist-github-teams.js")}"
-  order   = 20
-  enabled = true
-}
```

Comment:
What was the thinking here?

Reply:
Do we no longer need to whitelist the team, or is this done via the org script now?

Reply:
We do not use this rule, I think it was in place when everything was locked down to our team. It's not defined on the tenant.

Reply:
:+1: okay, thanks.